src/features/defi/providers/idle/components/IdleManager/Withdraw/IdleWithdraw.tsx

```diff
@@ -81,6 +81,7 @@ export const IdleWithdraw: React.FC<WithdrawProps> = ({ accountId }) => {
   useEffect(() => {
     ;(async () => {
       try {
+        if (state.userAddress && state.opportunity) return
```

Comment:
Note l.88 will go away with multi account feature, all we need is an `AccountId` / `AccountNumber`, and this is not the place to get/derive it